dependencies/FindO2GPU.cmake

```diff
@@ -100,7 +100,7 @@ if(ENABLE_CUDA)
   if(CMAKE_CUDA_COMPILER)
     # Forward CXX flags to CUDA C++ Host compiler (for warnings, gdb, etc.)
     STRING(REGEX REPLACE "\-std=[^ ]*" "" CMAKE_CXX_FLAGS_NOSTD ${CMAKE_CXX_FLAGS}) # Need to strip c++17 imposed by alidist defaults
-    set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -Xcompiler \"${CMAKE_CXX_FLAGS_NOSTD}\" --expt-relaxed-constexpr --extended-lambda --allow-unsupported-compiler -Xptxas -v")
```

Comment:
Would be interesting to check if `--generate-line-info` can affect the performance of the code. I believe not, but should be tested.

Reply:
Ah, indeed, this is a leftover of the debugging. I will remove it for the time being.